src/certFetcher/literal.ts

```diff
@@ -0,0 +1,11 @@
+import type { ICertFetcher } from './ICertFetcher.js';
+
+export default class LiteralCertFetcher implements ICertFetcher {
+  // eslint-disable-next-line class-methods-use-this
+  async readCertificate() {
+    return {
+      cert: 'some hard coded cert file',
```

Comment:
do these values get replaced somewhere

Reply:
They get replace if you implement a literal certificate fetcher. This text is a "fake" certificate because you don't really want to put a cert here.